tests/test.py

```diff
@@ -0,0 +1,75 @@
+import sys
+import socket
+
+HOST = '127.0.0.1'
+PORT = 7777 # The port used by the server
+
+def recv_all(sock, n):
+    b = bytearray()
+    read = 0
+    while read < n:
+        tmp = sock.recv(n-read)
+        if tmp:
+            read += len(tmp)
+            b.extend(tmp)
+    return bytes(b)
+
+def expect_response(sock, expected):
+    global passedAll
+    data = recv_all(s, len(expected))
+    print(data.decode('utf-8'), end='')
```

Comment:
Instead of print statement lets use logger.

```
import logging

logger.info('log statement')
```

Reply:
Used logger for log messages

Reply:
Ok